api/v1/flow/flow.proto

```diff
@@ -472,3 +472,97 @@ message LostEvent {
     // PERF_EVENT_RING_BUFFER.
     google.protobuf.Int32Value cpu = 3;
 }
+
+// AgentEventType is the type of agent event. These values are shared with type
+// AgentNotification in pkg/monitor/api/types.go
+enum AgentEventType {
+    AGENT_EVENT_UNKNOWN = 0;
+    AGENT_STARTED = 1;
+    POLICY_UPDATED = 2;
+    POLICY_DELETED = 3;
+    ENDPOINT_REGENERATE_SUCCESS = 4;
+    ENDPOINT_REGENERATE_FAIL = 5;
```

Comment:
nit: I'm wondering if this should be `ENDPOINT_REGENERATE_FAILURE` instead (for failure as opposed to success).

Reply:
Since I somehow messed up the rebase on #14126 (CI tests are currently failing because rebasing the PR on `master` fails), I'll probably do another iteration, so will address this before that.